### ISSUE TITLE
Build issue on Windows

### ISSUE THREAD

**Issue by aaronhagopian** [lloeki/matterfront#66]
When trying to build I am getting the following error:

Downloading electron-v$npm_package_electronVersion-linux-ia32.zip
Error: GET https://github.com/atom/electron/releases/download/v$npm_package_electronVersion/electron-v$npm_package_electronVersion-linux-ia32.zip returned 404
[ [Error: GET https://github.com/atom/electron/releases/download/v$npm_package_electronVersion/electron-v$npm_package_electronVersion-linux-ia32.zip returned 404] ] undefined

seems like the $npm_package_electronVersion variable is not getting set correctly, ideas?

Thanks

**Comment by LongLiveCHIEF** [comment]
Something with windows doesn't consume these the correct way. We're working on it for the full 1.3 release.

**Comment by geekytime** [comment]
Yeah, I have the same issue. It seems to have something to do with how cmd.exe/git bash/npm work together to layer environment variables.

That giant command-line should probably be split out into a shell script anyway...

**Comment by asadpiz** [comment]
Ran into same issue, had to hardcode the **$npm_package_electronVersion** variable to "0.35.4" in package.json.

because of the value of variable: *_"electronVersion": "0.35.4" *_ in the same file.

**Comment by markspolakovs** [comment]
FYI, a workaround is to edit the `package.json` file and under `scripts` -> `build` change `$npm_package_productName` and `$npm_package_electronVersion` to `%npm_package_productName%` and `%npm_package_electronVersion%`.
